### ISSUE TITLE
Add some diagnostic plots

### ISSUE THREAD

**Issue by twshimwell** [mhardcastle/ddf-pipeline#26]
Maybe helpful to add (and define) some diagnostic plots that are produced during the pipeline run to e.g. solutions

**Comment by mhardcastle** [comment]
Yeah, solutions plotting could go in the quality pipeline. A bit of fiddling with the infrastructure of PlotSols.py would allow some simple diagnostic plots to be made. I can sort this out after my holidays.
